Packs/Pcysys/ReleaseNotes/1_1_0.md

```diff
@@ -0,0 +1 @@
+Updated for new Pentera branding
```

Comment:
@ShahafBenYakir Do we have a convention or a header for pack changes?

Reply:
@mchasepan Have we added version and RN on purpose? You want the user to get an "update available" notification?
No convention

Reply:
Yes, since its a name change.